eval/http.go

```diff
@@ -290,7 +292,49 @@ func ApplyResponseContext(ctx context.Context, body hcl.Body, beresp *http.Respo
 		return nil
 	}
 
-	return ApplyResponseHeaderOps(ctx, body, beresp.Header)
+	if err := ApplyResponseHeaderOps(ctx, body, beresp.Header); err != nil {
+		return err
+	}
+
+	content, _, _ := body.PartialContent(config.BackendInlineSchema)
+	for _, attr := range content.Attributes {
```

Comment:
`range` should be replaced with a simple attr lookup since we want just a specific one.